CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Add: Evaluate group entityNameExp with a context including measures (#1334)
```

Comment:
entityNameExp is already documented in https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md but there is not info about whcih values are included in context used.
Since this PR just meassures are also in context, before this PR just serv, subserv, id, type and static attrs

Reply:
The only reference into the documentation to the context is the following:
https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md?plain=1#L514-L522

>The agent provides additional information, hereafter referred as **context**, in order to be used to evaluate the
>expression. In all cases the following data is available to all expressions:
>
>-   `id`: device ID
>-   `entity_name`: NGSI entity Name (principal)
>-   `type`: NGSI entity type (principal)
>-   `service`: device service
>-   `subservice`: device subservice
>-   `staticAttributes`: static attributes defined in the device or config group

No further references (including measures) neither for attribute transformation nor for the other fields like entityNameExp

Reply:
Added doc about measures in context for entityNameExp in https://github.com/telefonicaid/iotagent-node-lib/pull/1389/commits/dd1888e0a25e61802481f246c467c23744f269bb